src/main/java/com/mashape/unirest/request/BaseRequest.java

```diff
@@ -52,7 +51,7 @@ protected BaseRequest() {
 		super();
 	}
 
-	public HttpResponse<String> asString() throws UnirestException {
+	public HttpResponse<String> asString() {
```

Comment:
If you wanted to make this as small a change as possible you could leave the throws. It doesn't hurt anything to declare what RuntimeExceptions you throw and I personally think it's odd that we don't do that more in Java. It's not functional but it is a form of documentation. Or at least it makes the PR smaller. I'll leave it up to the library maintainers to make the call through.